apps/sms/js/error_dialog.js

```diff
@@ -71,15 +90,18 @@
       } else {
         console.error('Settings unavailable');
       }
+
+      body = {
+        id: bodyL10nId,
+        args: messageBodyParams
+      };
     }
 
     Dialog.call(this, {
       title: prefix + 'Title',
-      body: {
-        id: prefix + 'Body',
-        args: messageBodyParams
-      },
-      options: dialogOptions
+      body: body || bodyL10nId,
+      options: dialogOptions,
+      classes: ['error-dialog-' + prefix, 'error-dialog', ...additionalClasses]
```

Comment:
yes, but I thought it could be useful in the future. Having them would let us have more targeted CSS selectors instead of overriding BB (when we can do it of course...)